.github/autoissuelabeler.yml

```diff
@@ -61,7 +61,7 @@
   notify: [loicmathieu,evanchooly]
 - labels: [area/openapi, area/smallrye]
   title: "(?i).*openapi.*"
-  description: "(?i).*openapi.*"
+  description: "(?i).*(openapi|swagger).*"
```

Comment:
Shouldn't you change the title regex too?

Reply:
Yes !! sorry. doing it now.